--- a/gridpack_generation.sh	2019-07-05 23:28:50.929166000 +0200
+++ b/../../../TopEFT/mcgeneration/gridpack_generation.sh	2019-07-06 00:07:14.081590000 +0200
@@ -79,9 +79,9 @@
     MGBASEDIR=mgbasedir
     
     MG_EXT=".tar.gz"
-    MG=MG5_aMC_v2.6.5$MG_EXT
-    MGSOURCE=https://cms-project-generators.web.cern.ch/cms-project-generators/$MG
-    
+    MG=MG5_aMC_v2.6.6$MG_EXT
+    #MGSOURCE=https://cms-project-generators.web.cern.ch/cms-project-generators/$MG
+    MGSOURCE=https://launchpad.net/mg5amcnlo/2.0/2.6.x/+download/$MG
     MGBASEDIRORIG=$(echo ${MG%$MG_EXT} | tr "." "_")
     isscratchspace=0
     
